src/sec-6-5-improper.xml

```diff
@@ -137,10 +126,7 @@
   <subsection>
     <title>Convergence and Divergence</title>
     <p>
-      Our work so far has suggested that when we consider a nonnegative function <m>f</m> on an interval <m>[1,\infty]</m>,
-      such as <m>f(x) = \frac{1}{x}</m> or <m>f(x) = \frac{1}{x^{3/2}}</m>,
-      there are at least two possibilities for the value of <m>\lim_{b \to \infty} \int_1^b f(x) \, dx</m>:
-      the limit is finite or infinite.
+      Activity 6.13 suggests that  <m>\lim_{b \to \infty} \int_1^b f(x) \, dx</m> is either finite or infinite (or it doesn't exist).
```

Comment:
There should be an `xref` here instead of a hard-coded reference to Activity 6.13. (In particular, I don't think there is an Activity 6.13 under the latest numbering scheme.)

Reply:
Thanks Mitch for pointing this out.  Fixed.